remill/Arch/X86/Runtime/Types.h

```diff
@@ -23,19 +23,19 @@ typedef union {
     uint8_t lsd:4;  // Least-significant digit
     uint8_t msd:4;  // Most-significant digit
   } __attribute((packed)) pair;
-} __attribute((packed)) dec_digit_pair_t;
+} __attribute((packed)) bcd_digit_pair_t;
```

Comment:
Can you make this a `union bcd_digit_pair` or `union bcd_digit_pair_t`. I don't like `typedef`s on `struct` or `union` definitions because then they cannot be as easily forward-declared (not relevant here, but just as a general practice).

Reply:
Good call, on a second glance, I prefer that too.

Reply:
Addressed: 0bdadea592811dd957fc5a94e1343e36ca51566d